crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -645,17 +682,21 @@ impl Entities {
         self.len = count as u32;
     }
 
-    /// Accessor for getting the length of the vec in `self.meta`
+    /// The count of all entities in the [`World`], living and dead.
```

Comment:
Does this count reserved entities too?

Reply:
Fixed up this comment to be a bit more descriptive, and using the terms that are more in alignment with the rest of the docs.